src/components/index-page/IndexLaunched.tsx

```diff
@@ -8,16 +8,19 @@ export const IndexLaunched = () => (
   <section className="index-launched">
     <div className="index-launched__contents">
       <div className="index-launched__heading-container">
-        <h2 className="index-launched__heading">Go from stuck and stalled out to LAUNCHED</h2>
+        <h2 className="index-launched__heading">
+          Go from stuck and stalled out to{" "}
+          <span className="index-launched__uppercase">launched</span>
+        </h2>
       </div>
       <div className="index-launched__columns-container">
         <div className="index-launched__column">
           <div className="index-launched__image-container">
             <img className="index-launched__image" src="https://placehold.co/378x330" />
           </div>
-          <h2 className="index-launched__column-heading">
+          <h4 className="index-launched__column-heading">
```

Comment:
we should avoid skipping numbers and make these `<h3>` elements to semantically define an outline format for the markup